core/domain/exp_domain.py

```diff
@@ -2030,12 +2030,19 @@ def _verify_all_states_reachable(self) -> None:
                     all_outcomes = curr_state.interaction.get_all_outcomes()
                     for outcome in all_outcomes:
                         dest_state = outcome.dest
+                        dest_if_stuck_state = outcome.dest_if_really_stuck
                         if (
                             dest_state is not None and
                             dest_state not in curr_queue and
                             dest_state not in processed_queue
                         ):
                             curr_queue.append(dest_state)
+                        if (
+                            dest_if_stuck_state is not None and
+                            dest_if_stuck_state not in curr_queue and
+                            dest_if_stuck_state not in processed_queue
+                        ):
+                            curr_queue.append(dest_if_stuck_state)
```

Comment:
Is it possible that `dest_if_stuck_state` and `dest_state` are equal? if so, should we make sure the curr_queue only has one copy? Not sure

Reply:
If the dest_if_stuck_state and dest_state are equal, since dest_state will be pushed in the curr_queue first, the dest_if_stuck_state will not be pushed since there is already an instance of it in the curr_queue ig.